operatorconfig/clientconfig/apexconnectivityclient/v1.0.0/statefulset.yaml

```diff
@@ -60,7 +60,7 @@ rules:
     verbs: ["list", "watch", "get"]
   - apiGroups: [""]
     resources: ["secrets"]
-    verbs: ["list","get", "create", "update"]
+    verbs: ["list","get", "create", "update", "delete"]
   - apiGroups: [""]
     resources: ["configmaps"]
     verbs: ["get", "create", "delete", "update"]
```

Comment:
How come we have list permissions for secrets but not for config maps?

Reply:
Tried deleting the license secrets created on the cluster and it worked fine.